README.md

```diff
@@ -293,6 +293,8 @@ overridden through configuration.
 | PGX_PG_CONFIG_VERSION                       | Provided by the running Postgres instance                                     | Used by `pgx` to build the PL/Rust user function.                                                                                                                                                                     |
 | PGX_PG_CONFIG_CPPFLAGS                      | Provided by the running Postgres instance                                     | Used by `pgx` to build the PL/Rust user function (technically unused by PL/Rust's build process as PL/Rust does not include the pgx "cshim" for which this is normally used).                                         |
 | PGX_PG_CONFIG_INCLUDEDIR-SERVER             | Provided by the running Postgres instance                                     | Used by `pgx` to build the PL/Rust user function.                                                                                                                                                                     |
+| PLRUSTC_USER_CRATE_NAME                     | The name of the user crate                                                    | Used by `plrust` to help `plrustc` distinguish between the PL/Rust user function and dependencies.                                                                                                                    |
+| PLRUSTC_USER_CRATE_MAY_ACCESS               | The user crate dir and the cargo target dir, separated by `:`.                | When compiling the user crate (given by `PLRUSTC_USER_CRATE_NAME`), `plrustc` ensures no source files outside of these (`:`-delimited) directories are accessed. Every path in the list should be absolute and have symlinks resolved. The sysroot is currently automatically added to this list. |
```

Comment:
For a hot second I thought this was going to be a bool, while reading the rest of the code. Could this name suggest the PATH-like nature of it?